File: gang/week11/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int i = s.length() / 2;
        if(s.length() % 2 == 0) {   // 길이가 짝수인 경우
            answer = s.substring(i-1,i+1);  // 길이의 절반-1과 +1의 값을 가져옴
        }
        else {  // 홀수인 경우
            answer = s.substring(i,i+1);    // 길이의 절반+1의 값을 가져옴
        }
        return answer;
    }
}

File: jeonghyeon/11week/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    if(s.length % 2 == 0) { // 짝수 개수면 가운데 두개 뽑기
        answer = s[s.length / 2 - 1] + s[s.length / 2];
    } else { // 홀수 개수면 가운데 한개
        answer = s[Math.floor(s.length / 2)];
    }
    return answer;
}